src/recvec.h

```diff
@@ -240,7 +241,7 @@ namespace libsemigroups {
       }
     }
 
-   private:
+   protected:
```

Comment:
Wouldn't it be preferable to leave this `private` and to use the member functions in the `Graph` class?